store/psql/queries/reward_era_seq_insert.sql

```
@@ -12,4 +12,7 @@ INSERT INTO reward_era_sequences (
 )
 VALUES @values
 
-ON CONFLICT (era, stash_account, validator_stash_account, kind) DO NOTHING;
+ON CONFLICT (era, stash_account, validator_stash_account, kind)
```

is that index exists for now?  it might silently errors now because of the do nothing clause.  If not you may like to add its creation to the migration

---

```
CREATE UNIQUE INDEX idx_rewards_accounts_kind
    ON reward_era_sequences(era, stash_account, validator_stash_account, kind);
```

https://github.com/figment-networks/polkadothub-indexer/blob/master/migrations/000018_create_reward_era_seqs_table.up.sql#L22

:D